libs/types/src/lib/types.ts

```diff
@@ -78,3 +78,8 @@ export interface IRefreshResponse {
   ok: boolean;
   accessToken: string;
 }
+
+export interface IRoutes {
```

Comment:
It's an interface defining a route, so:
```suggestion
export interface IRoute {
```